config/kafka-monitor.properties

```diff
@@ -111,6 +111,16 @@
 #  }
 
 
+#  Example kafka-service to report metrics 
+#  "reporter-kafka-service": {
+#    "class.name": "com.linkedin.kmf.services.KafkaMetricsReporterService",
+#    "report.interval.sec": 3,
+#    "zookeeper.connect": "localhost:2181",
```

Comment:
Can we add `report.metrics.list` in this example?

Reply:
Okay, I'll modify the sample like this:
```
#  Example kafka-service to report metrics 
#  "reporter-kafka-service": {
#    "class.name": "com.linkedin.kmf.services.KafkaMetricsReporterService",
#    "report.interval.sec": 3,
#    "zookeeper.connect": "localhost:2181",
#    "bootstrap.servers": "localhost:9092",
#    "topic": "kafka-monitor-topic-metrics",
#    "report.metrics.list": [
#    "kmf:type=kafka-monitor:offline-runnable-count",
#    "kmf.services:type=produce-service,name=*:produce-availability-avg",
#    "kmf.services:type=consume-service,name=*:consume-availability-avg",
#    "kmf.services:type=produce-service,name=*:records-produced-total",
#    "kmf.services:type=consume-service,name=*:records-consumed-total",
#    "kmf.services:type=consume-service,name=*:records-lost-total",
#    "kmf.services:type=consume-service,name=*:records-duplicated-total",
#    "kmf.services:type=consume-service,name=*:records-delay-ms-avg",
#    "kmf.services:type=produce-service,name=*:records-produced-rate",
#    "kmf.services:type=produce-service,name=*:produce-error-rate",
#    "kmf.services:type=consume-service,name=*:consume-error-rate"
#    ]
#  }
```